src/content/docs/security/security-privacy/information-security/security-bulletins.mdx

```diff
@@ -43,6 +43,20 @@ Security bulletins for our [APM](/docs/apm/new-relic-apm/getting-started/introdu
   </thead>
 
   <tbody>
+    <tr>
+       . <td>
+        [NRSG22-01](/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01)
+       </td>
+
+       <td>
+        New Relic recommends that customers who deploy the .NET Agent in a configuration using Microsoft Extensions Logging (MEL) should update to version 10.1.0 or later.
```

Comment:
`Agent` s/be lowercase (not a release blocker)

Reply:
Interesting, this is showing as `agent` in VS code.